shared/src/types/key/ed25519.rs

```diff
@@ -27,6 +29,15 @@ pub struct PublicKey(ed25519_dalek::PublicKey);
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SecretKey(ed25519_dalek::SecretKey);
 
+/// Ed25519 keypair
+#[derive(Debug, Serialize, Deserialize, BorshSerialize, BorshDeserialize)]
+pub struct Keypair {
```

Comment:
We added this newtype for Keypair, because initially we were serializing the wallet with borsh into binary. Now that the wallet is using TOML, it's no longer needed, but it still helpful in some cases as we also have newtype wrappers for pk, sk and sig, so there're less conversions where we use these.